src/cryptography/hazmat/backends/__init__.py

```diff
@@ -18,7 +18,10 @@ def default_backend():
     return _default_backend
 
 
-def _get_backend(backend):
+T = typing.TypeVar("T")
+
+
+def _get_backend(backend: typing.Optional[T]) -> T:
```

Comment:
Can this just be `typing.Optional[Backend] -> Backend` now?

Reply:
Yeah that should be fine now since we're not unwrapping subtypes.

Reply:
Great, let's do that. Should also remove the need to type a bunch of local variables.